src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.csproj

```diff
@@ -122,6 +122,12 @@
       <IlmergeCommand>$(IlmergeCommand) ^
         $(PathToBuiltNuGetPack) ^
         @(BuildArtifacts, ' ')</IlmergeCommand>
+      <IlmergeCommand Condition=" '$(IsCore)' == 'true' ">$(IlmergeCommand) ^
+        /lib:$(PkgMicrosoft_Build_Utilities_Core)\lib\netstandard2.0 ^
```

Comment:
I'm not sure I understand why we need to ilmerge these. 
Were they being ilmerged before?

Reply:
In short, yes. 
This change does not make anything new ILMerged. All this does is point to folders (libs) where assemblies can be referenced.